BOZ_App/resources/lang/nl.json

```diff
@@ -0,0 +1,3 @@
+{
+    "De maatschappij is het probleem, niet de mens met de beperking": "De maatschappij is het probleem, niet de mens met de beperkin"
```

Comment:
Ik neem aan dat dit beperking moet zijn ipv beperkin

Reply:
Klopt. Zal ik fixen in mijn implementatie in de front end er van